packages/riverpod_generator/README.md

```diff
@@ -156,6 +156,21 @@ class Home extends ConsumerWidget {
 }
 ```
 
+## Global configuration
```

Comment:
I don't like this being in the root `examples` folder.
it probably should be in the riverpod_generator folder

Reply:
that makes more sense. I've just changed